File: observer/Samuel Sales/DesafioObserver_Samuel/src/Canais/GloboSat.java
package Canais;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class GloboSat implements ObserverGrupoGlobo {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Canal Globo Sat");
    }

}

File: observer/PadraoObserverII/PadraoObserverII/Canais/GloboSat.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Canais
{
    class GloboSat : IObserverCanal
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Canal Globo SAT!");
        }
    }
}
